src/containers/join-dao.js

```diff
@@ -62,7 +69,7 @@ const mapDispatchToProps = (dispatch, ownProps) => {
       dispatch(resetNewProposal())
       
       let semBalance = await SemadaCore.getSemBalance(publicAddress)
-      dispatch(saveSemBalance(publicAddress))
+      dispatch(saveSemBalance(semBalance))
```

Comment:
SEM is already being reduced in the API/semada core. We shouldn't be making another call to do this here. Imagine that we were using Ethereum, there wouldn't be a call to save SEM balance. Instead we should be retrieving the SEM balance from the API from app wrapper when it refreshes data.